src/components/notification/notification.module.scss

```
@@ -0,0 +1,143 @@
+
+.notification {
+  --notification-bg: #{$color-info-soft};
+  --notification-fg: #{$color-info-soft-foreground};
+  --notification-border: #{$color-info-soft-border};
+
+  display: flex;
+  align-items: center;
+  gap: $space-3;
+  padding: $space-3 $space-4;
+
+  background-color: var(--notification-bg);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)